UnityProject/Assets/Scripts/FluidPipe/PipeFunctions.cs

```diff
@@ -333,6 +333,8 @@ public void EqualiseWithOutputs(List<PipeData> others)
 			float density = this.Density();
 			foreach (var Pipe in others)
 			{
+				if(Pipe == null) continue;
+
```

Comment:
NullReferenceException: Object reference not set to an instance of an object
Pipes.PipeFunctions.PipeOrNet (Pipes.PipeData pipe) (at /github/workspace/UnityProject/Assets/Scripts/FluidPipe/PipeFunctions.cs:168)
Pipes.MixAndVolume.EqualiseWithOutputs (System.Collections.Generic.List`1[T] others) (at /github/workspace/UnityProject/Assets/Scripts/FluidPipe/PipeFunctions.cs:336)
Pipes.ReservoirAction.TickUpdate () (at /github/workspace/UnityProject/Assets/Scripts/FluidPipe/PipeActions.cs:47)
Pipes.PipeData.TickUpdate () (at /github/workspace/UnityProject/Assets/Scripts/FluidPipe/PipeData.cs:276)
AtmosManager.DoTick () (at /github/workspace/UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosManager.cs:86)
AtmosManager.Update () (at /github/workspace/UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosManager.cs:73)